build/BUILD.bazel

```diff
@@ -36,6 +36,25 @@ install_lualib_deps_cmd = "\n".join([
     for dep in lualib_deps
 ])
 
+wasmx_deps = select({
+    "@kong//:wasmx_flag": [
+        "@ngx_wasm_module//:lua_libs",
+    ],
+    "//conditions:default": [],
+})
+
+wasmx_lua_libs_cmd = select({
+    "@kong//:wasmx_flag": """
+    for fname in $(locations @ngx_wasm_module//:lua_libs); do
+        base=${fname##*/ngx_wasm_module/lib/}
+        dest="${BUILD_DESTDIR}/openresty/lualib/$base"
+        mkdir -p "$(dirname "$dest")"
+        cp -P --preserve=all "$fname" "$dest"
```

Comment:
this _might_ not work under macos 🤔

Reply:
hmm, good call; I think we can probably get away with dropping the flags for the time being